policy-controller/k8s/status-controller/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+use linkerd_policy_controller_k8s_api as k8s;
+use linkerd_policy_controller_k8s_index as index;
+use tokio::sync::mpsc;
+
+pub async fn process_patches(
```

Comment:
nit: maybe worth sticking a 
```suggestion
#[tracing::instrument(level = "info", skip(client, patches_rx))]
pub async fn process_patches(
```
on this?